keysign/bluetoothoffer.py

```diff
@@ -89,9 +89,8 @@ def allocate_code(self):
             return None
         if self.server_socket is None:
```

Comment:
since we're using the `code` from the `try` block, make it an `else` here.  If we ever change the `except` block to not return, we might get surprised.

Reply:
I changed this with the PR #55 

Maybe I should merge this PR in the #55 ?